documentation/asciidoc/accessories/display/display_intro.adoc

```diff
@@ -1,47 +1,78 @@
 == Raspberry Pi Touch Display
 
-The https://www.raspberrypi.com/products/raspberry-pi-touch-display/[Raspberry Pi Touch Display] is an LCD display which connects to the Raspberry Pi through the DSI connector. In recent versions of Raspberry Pi OS, both the HDMI and LCD display outputs can be used at the same time.
+The https://www.raspberrypi.com/products/raspberry-pi-touch-display/[Raspberry Pi Touch Display] is an LCD display which connects to the Raspberry Pi through the DSI connector. You can also use the HDMI and DSI display outputs at the same time.
 
 === Board Support
 
-The DSI display is designed to work with all models of Raspberry Pi, however early models that do not have mounting holes (the Raspberry Pi 1 Model A and B) will require additional mounting hardware to fit the HAT-dimensioned stand-offs on the display PCB.
+The Touch Display is designed to work with all models of Raspberry Pi, however early models without mounting holes (the Raspberry Pi 1 Model A and B) will require additional mounting hardware to fit the HAT-sized stand-offs on the display PCB.
 
-=== Physical Installation
+=== Connecting a Raspberry Pi
 
-The following image shows how to attach the Raspberry Pi to the back of the Touch Display (if required), and how to connect both the data (ribbon cable) and power (red/black wires) from the Raspberry Pi to the display. If you are not attaching the Raspberry Pi to the back of the display, take extra care when attaching the ribbon cable to ensure it is the correct way round. The black and red power wires should be attached to the GND and 5V pins respectively.
+The following image shows how to attach a Raspberry Pi directly to the back of the Touch Display using its standoffs. It also shows the connections you need to make between Raspberry Pi and the display's PCB. You can also mount the Touch Display in a separate chassis if you have one available. The connections will remain the same, though you may need longer cables depending on the chassis you use.
```

Comment:
In the same way I don't like to use "here" as an anchor, I try and steer clear of language like "The following image shows" and keep the text independent of the illustrations. Illustrations should add, not be essential, to understanding the narrative text.